script/Arcueid/Day 8/08_00_ARC08_8_2.txt

```diff
@@ -151,17 +151,18 @@ In Judo, you'd call it an osotogari.
 [sha:2f801b5a4569c353ef2c5ca88bd95dffa5a91de6]{
 -- Page 592, Offset 21271.
 -- 背中から地面に叩きつけられる。
-I am thrown onto the ground from behind.
+I'm thrown onto the ground from behind.
 }
 [sha:7e1fcbf79cf05dc9d1037c513f235e9195793fb3]{
 -- Page 592, Offset 21272.
 -- 先ほどの転倒もどきとは訳が違う。
-It is completely different from the falling motion from before.
+It's completely different from the previous falling motion.
 }
 [sha:fba2f401bf8593e956a66bcacfe8a8b70cae77ad]{
 -- Page 592, Offset 21273.
 -- 地面に倒れるだけでも、確かな腕力で地面に<落|・><と|・><さ|・><れ|・><る|・>と人体は容易に破壊される。
-Although I'm simply falling to the ground, a human body can easily be destroyed if one gets dropped with precise arm strength.
+// not quite sure what form of strength specifically 腕力 is because there are lots of results
```

Comment:
I think this is basically right, but this might be above my Japanese pay grade. You should ask Gekimon or Petrikow.